### PR TITLE
Add version to generated package based on version of e-Crypt

### LINKED ISSUE #14: Add version to generated self-extracted package
Currently if we use e-crypt to generate self-extracted package we can generate an executable but generated executable does not contains version in properties.
So it is impossible to figure out with which version of e-crypt certain package was created.

### PR DESCRIPTION
**Pull request by iarovyi**
fix #14

### CODE REVIEW

**Comment by iarovyi** [comment]
@antondimitrov @omer-demir
Please review pull request

File: src/eCrypt/Options.cs [hunk 1]
```diff
@@ -23,6 +23,9 @@ internal class Options
         [Option('t', "target", Required = true, HelpText = "Folder of file that should be encrypted and embedded into generated executable")]
         public string EncryptionTargetPath { get; set; }
 
+        [Option('c', "content-version", Required = false, HelpText = "Version of the content to be encrypted. If provided it will be used for 'file version' property of generated executable")]
```
Comment (antondimitrov) [inline src/eCrypt/Options.cs]: `-v` would be the preferred name

Comment (antondimitrov) [inline src/eCrypt/Options.cs]: I believe we will need extra validation to ensure the format is matching .net `version` class e.g. `1.1.1.1`

**Review by antondimitrov — CHANGES_REQUESTED** [review]

**Comment by iarovyi** [comment]
capture

**Comment by Eugene1982** [comment]
PR was tested, @antondimitrov please merge

**Comment by antondimitrov** [comment]
@iarovyi Can you please squash appropriately?

**Comment by iarovyi** [comment]
@antondimitrov Squashed commits

**Review by antondimitrov — APPROVED** [review]

**Pull request merged**